docs/modules/reference/pages/service-assurance/monitors/PtpMonitor.adoc

```diff
@@ -0,0 +1,60 @@
+
+= PtpMonitor
+:description: Learn how to configure and use the PtpMonitor in OpenNMS {page-component-title} to monitor the correct PTP port state for an interface.
+
+The Precision Time Protocol allows the precise synchronization of clocks in measurement and control systems implemented with packet-based networks.
+OpenNMS can monitor the port state of SNMP-enabled interfaces by using the PTP MIB defined in RFC 8173.
```

Comment:
```suggestion
The PTP monitor allows you to test the port state of SNMP-enabled interfaces using the PTP MIB defined in RFC 8173.
```

Reply:
Just to remove the problem with "OpenNMS" as a project, vs. products such as Horizon, Meridian, ${powered-by-whatever}.